packages/backend/src/managers/playgroundV2Manager.ts

```diff
@@ -232,6 +264,9 @@ export class PlaygroundV2Manager implements Disposable {
     }
 
     this.#conversationRegistry.completeMessage(conversationId, messageId);
+    this.telemetry.logUsage('playground.message.complete', {
+      duration: Date.now() - start,
+    });
```

Comment:
probably a later improvement

but maybe we should log some contextual information on each event.

For example what is the anonymized version of the model name (like its sha), some other useful parameters 
like the number of cpu/memory of the machine, etc